charts/optimize-live/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.3.0
+version: 0.3.1
```

Comment:
is comment accurate for our versioning strategy?

if semver & bumping each time, i'm guessing this should be 0.4.0.

Reply:
also i'm guessing controller image tag would need to be updated along with this PR